test/Typewriter.Test/TestDataJava/com/microsoft/graph/requests/extensions/IEntityType3ForwardRequestBuilder.java

```diff
@@ -28,4 +28,5 @@
      * @return the IEntityType3ForwardRequest instance
      */
     IEntityType3ForwardRequest buildRequest(final java.util.List<? extends com.microsoft.graph.options.Option> requestOptions);
+
```

Comment:
Can we add a composable function sample to the test data?

Reply:
done!